examples/Podfile.lock

```diff
@@ -15,4 +15,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f68cc4584957218d15c617848be26ebbd4bf5c8f
 
-COCOAPODS: 1.7.2
+COCOAPODS: 1.8.4
```

Comment:
Is this update required? I didn't see the CocoaPods version change listed in the PR summary.